index.c

```diff
@@ -2183,6 +2194,7 @@ int mutt_index_menu(void)
       {
         struct Mailbox *m = NULL;
         OptNews = false;
+        char *cp = NULL;
```

Comment:
This initialisation needs to be moved below. It is within and `#ifdef` block, but `cp` is used outside the `#ifdef`. If this is to scope a NNTP specific `cp` variable, we should add a comment.